src/index.html

```diff
@@ -10,23 +10,34 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search__form"
+      data-qa="big"
+    >
```

Comment:
The `<form>` tag is self-closing, which is incorrect in HTML. It should be opened with `<form>` and closed with `</form>`.